mm2src/coins_activation/src/eth_with_token_activation.rs

```diff
@@ -290,6 +290,7 @@ impl PlatformCoinWithTokensActivationOps for EthCoin {
             platform_conf,
             activation_request.platform_request,
             priv_key_policy,
+            // ChainSpec
```

Comment:
what's that?

Reply:
my own helper comment (to not keep everything in the head), will remove it

Reply:
removed https://github.com/KomodoPlatform/komodo-defi-framework/pull/2467/commits/1f69c9d0e62b5e8f996f29f04c50a51a2e6baec8